fastn-js/js/dom.js

```diff
@@ -2,6 +2,12 @@ let fastn_dom = {};
 
 fastn_dom.styleClasses = "";
 
+fastn_dom.InternalClass = {
+    FT_COLUMN: 'ft_column',
+    FT_ROW: 'ft_row',
+    FULL: "full",
```

Comment:
Do you think `full` is a good name? Can you think some better name then this? @harshdoesdev @Heulitig

Reply:
We think `ft_full_size` is a better name

Reply:
Go ahead with this @harshdoesdev @Heulitig